content/troubleshooting/scaledobject-not-scale.md

```diff
@@ -1,14 +1,36 @@
 +++
-title = "Why is my `ScaledObject` paused?"
+title = "Why is my `ScaledObject` not scaling as expected?"
```

Comment:
I personally would prefer a seperate item for this; people using pausing will more likely be helped faster if we have a dedicated item.

You can still refer to the new item from within the paused entry

Reply:
Agree a separate section would be great